### PR TITLE
feat: add article about conditionally exported dev tools

### PR DESCRIPTION
**Pull request by stepan662**

### CODE REVIEW

File: blog/2023-02-12-conditional-export.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+---
+slug: conditional-export
+title: 'NPM Packge: How to Conditionally Include Developer Tools?'
+authors: [sgranat]
+image: '/img/blog/free-localization/painless_localization.png'
+description: 'Creating a package with smaller bundle size for production by excluding developer tools'
+tags: [localization, i18n, translation]
+---
+
+Reducing the size of your production bundle can be a significant challenge. One common issue is including developer tools in the package, they are essential in development mode but add unnecessary bulk to the production bundle. In this article, we'll explore a solution to this problem that involves including the developer tools conditionally, based on the environment in which the package is being used.
```
Comment (TolgeeMachine) [inline blog/2023-02-12-conditional-export.md:10]: To the intro I would add some references to other similar use cases, like react query, react dev tools, redux etc. To show, that this is not just "Look how cool we are", but show that this has use cases also in outside world.

File: blog/2023-02-12-conditional-export.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+---
+slug: conditional-export
+title: 'NPM Packge: How to Conditionally Include Developer Tools?'
+authors: [sgranat]
+image: '/img/blog/free-localization/painless_localization.png'
+description: 'Creating a package with smaller bundle size for production by excluding developer tools'
```
Comment (marketachalupnikova) [inline blog/2023-02-12-conditional-export.md:6]: Please write longer meta description: 110 to 160 characters, now it's 87

File: blog/2023-02-12-conditional-export.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+```json
+{
+  "exports": {
+    "import": "./dist/bundle.esm.js",
+    "require": "./dist/bundle.cjs.js"
+  }
+}
+```
+
+Modern build tools utilize this feature by using semi-standard fields "production" and "development", so we can do:
```
Comment (TolgeeMachine) [inline blog/2023-02-12-conditional-export.md:101]: Can you specify which modern tools you have in mind specifically.

**Review by TolgeeMachine — COMMENTED** [review]
Good job! 🎖️

**Review by marketachalupnikova — CHANGES_REQUESTED** [review]

File: blog/2023-02-12-conditional-export.md (hunk 3 above)
Comment (stepan662) [reply blog/2023-02-12-conditional-export.md:103]: I'm mentioning VITE in few sentences before, I think it's not necessary to write it again

File: blog/2023-02-12-conditional-export.md (hunk 1 above)
Comment (stepan662) [reply blog/2023-02-12-conditional-export.md]: Ok

**Review by marketachalupnikova — APPROVED** [review]

**Pull request merged**